### PR TITLE
Update package.json to fix Scalingo support.

### PR DESCRIPTION
**Pull request by kalhel**
at scalingo deployment show this error:
Build complete, shipping your container... Waiting for your application to boot... ! Error deploying the application ! → Your application has stopped unexpectedly with the following output:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "watch": "grunt default",
     "lint": "grunt lint",
     "grunt": "grunt"
+    "postinstall": "npm install && npm run build"
```
Comment (pogo-excalibur) [inline package.json]: You've missed out a comma between this line and the previous one.

**Comment by pogo-excalibur** [comment]
You've called the npm task which does the install, inside the install process. This is causing a recursive loop.

Remove the `npm install &&` part from the `postinstall` task.

It would be helpful if you could post the rest of the error in the description above.

**Comment by pogo-excalibur** [comment]
@kalhel Is this still something that you're interested in tackling?

**Comment by sebastienvercammen** [comment]
PR closed because the author has been unresponsive for ~2 weeks.

**Pull request closed without merging**